eth-providers/src/base-provider.ts

```diff
@@ -744,7 +767,7 @@ export abstract class BaseProvider extends AbstractProvider {
 
     // to and contractAddress may be undefined
     return {
-      confirmations: 1,
+      confirmations: DEFAULT_CONFIRMATIONS,
```

Comment:
should be the confirmation of the block. i.e. latest block number minus query block number